lighthouse-extension/app/popup.html

```diff
@@ -38,15 +38,36 @@ <h2 class="header-titles__url">...</h2>
 
   <main class="main" role="main">
     <div class="feedback"></div>
-    <button class="generate-report">Generate report</button>
+
+    <button class="button button--link button--configure" id="configure-options">Options</button>
+    <button class="button button--generate" id="generate-report">Generate report</button>
   </main>
 
-  <aside class="status">
+  <aside class="status subpage">
     <div class="status__spinner"></div>
     <div class="status__msg">Starting...</div>
     <div><small class="status__detailsmsg"></small></div>
   </aside>
 
+  <aside class="options subpage">
+    <h2 class="options__title">Options</h2>
+    <ul class="options__list">
+        <li><input type="checkbox" id="offline" checked="checked" value="App can load on offline/flaky connections" /><label for="offline">App can load on offline/flaky connections</label></li>
```

Comment:
was thinking the same but than I need something like posthtml? Or should I do a simple replace string ?